### PR TITLE
ENG-7634: Updates pipeline example

### PR DESCRIPTION
**Pull request by bredamc**
ENG-7634: Updates pipeline example

### CODE REVIEW

**Comment by bredamc** [comment]
@diegolovison Updated as you requested, please review. 

Please also advise whether we should update line 175 as follows:
from: `ray_fn(openshift_server=openshift_server, openshift_token=openshift_token)`
to: `ray_fn()`

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 1]
```diff
@@ -110,34 +110,30 @@ from kfp import dsl
...
-   auth = TokenAuthentication( <2>
-       token=openshift_token, server=openshift_server
-   )
-   auth_return = auth.login()
-
-
-   cluster = Cluster( <3>
+   cluster = Cluster( <4>
        ClusterConfiguration(
+           namespace="my_project", <5>
```
Comment (diegolovison) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:121]: Can be removed

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 2]
```diff
@@ -110,34 +110,30 @@ from kfp import dsl
...
+           local_queue="local_queue_name", <7>
        )
    )
 
 
    print(cluster.status())
-   cluster.up() <6>
-   cluster.wait_ready() <7>
+   cluster.up() <8>
+   time.sleep(180) <9>
```
Comment (diegolovison) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:136]: Should we say the reason behind time.sleep? that is a kwon issue

**Review by diegolovison — CHANGES_REQUESTED** [review]
Requested changed because of the project namespace

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc (hunk 1 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:121]: Updated instead to indicate that it's optional, as agreed in Slack.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc (hunk 2 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:137]: Added for now, but we don't usually document known issues in the main user doc.

**Comment by bredamc** [comment]
@diegolovison Ready for review and approval.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 3]
```diff
@@ -156,49 +153,48 @@ def ray_fn(openshift_server: str, openshift_token: str):
...
-<5> Specifies the local queue to which the Ray cluster will be submitted: you can omit this line if you configured a default local queue
-<6> Creates a Ray cluster using the specified image and configuration
-<7> Waits for the Ray cluster to be ready before proceeding
-<8> Enables the Ray client to connect to a secure Ray cluster that has mutual Transport Layer Security (mTLS) enabled; mTLS is enabled by default in the CodeFlare component in {productname-short}
-<9> Replace the example details in this section with the details for your workload
-<10> Removes the Ray cluster when your workload is finished
-<11> Replace the example name and description with the values for your workload
-<12> Compiles the Python code and saves the output in a YAML file
+<1> Imports Ray.
+<2> Imports the `time` package so that you can use the `sleep` function to wait during code execution, as a workaround for link:https://issues.redhat.com/browse/RHOAIENG-7346[RHOAIENG-7346].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:185]: Note that 7346 is due to be fixed at 2.11. After it is fixed, this reference will need to be removed.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 4]
```diff
@@ -156,49 +153,48 @@ def ray_fn(openshift_server: str, openshift_token: str):
...
+<3> Imports from the CodeFlare SDK the packages that define the cluster functions.
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:186]: Would it be clearer to reword this to put what is imported first?

"Imports packages that define cluster functions from the CodeFlare SDK"

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 5]
```diff
@@ -156,49 +153,48 @@ def ray_fn(openshift_server: str, openshift_token: str):
...
+<4> Specifies the Ray cluster configuration: replace these example values with the values for your Ray cluster.
+<5> Optional: Specifies the project where the Ray cluster is created. Replace the example value with the name of your project. If you omit this line, the Ray cluster is created in the current project.
+<6> Specifies the location of the Ray cluster image. If using a disconnected environment, replace the default value with the location for your environment.
+<7> Specifies the local queue to which the Ray cluster will be submitted. If a default local queue is configured, you can omit this line.
+<8> Creates a Ray cluster by using the specified image and configuration.
+<9> Waits for the Ray cluster to be ready before proceeding. As a workaround for link:https://issues.redhat.com/browse/RHOAIENG-7346[RHOAIENG-7346], use `time.sleep(180)` instead of `cluster.wait_ready()`.
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:192]: Could a better term be used than "to be ready"?

"To initialize"?

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 6]
```diff
@@ -156,49 +153,48 @@ def ray_fn(openshift_server: str, openshift_token: str):
...
-<9> Replace the example details in this section with the details for your workload
-<10> Removes the Ray cluster when your workload is finished
-<11> Replace the example name and description with the values for your workload
-<12> Compiles the Python code and saves the output in a YAML file
+<1> Imports Ray.
+<2> Imports the `time` package so that you can use the `sleep` function to wait during code execution, as a workaround for link:https://issues.redhat.com/browse/RHOAIENG-7346[RHOAIENG-7346].
+<3> Imports from the CodeFlare SDK the packages that define the cluster functions.
+<4> Specifies the Ray cluster configuration: replace these example values with the values for your Ray cluster.
+<5> Optional: Specifies the project where the Ray cluster is created. Replace the example value with the name of your project. If you omit this line, the Ray cluster is created in the current project.
+<6> Specifies the location of the Ray cluster image. If using a disconnected environment, replace the default value with the location for your environment.
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:189]: The sentence beginning with "If using..".

How about "If you use..."?

**Review by chtyler — CHANGES_REQUESTED** [review]
Checked the Asciidoc code and rendered output. A few minor suggestions made. Checked the code blocks in the rendered output for consistency.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc (hunk 3 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:185]: Follow-on task created to revert the code and callouts when 7346 is fixed.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc (hunk 4 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc]: Rephrased as follows:
"Imports packages from the CodeFlare SDK to define the cluster functions."

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc (hunk 5 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc]: I believe the `ray.init` command does the initialization.
This callout is for the original `cluster.wait_ready() ` command, so I think the current text is more accurate. I've rephrased slightly as follows:
" Waits until the Ray cluster is ready before proceeding."

**Review by chtyler — APPROVED** [review]
